### DISCUSSION TITLE
Cannot use native find in new code view because text is lazy-loaded

### DISCUSSION THREAD

**Discussion by saagarjha** [community/community#42773] [Repositories]
### Select Topic Area

Bug

### Body

In the old code view I could hit ⌘F and the native browser find would function as expected. In the new one the source code is not loaded on the page so it will often return "0 results" when the text actually exists on the page. I know that the new code view has its own find experience, but it's often not what I want because navigating between results is much slower or very difficult (especially on mobile).

**Comment by jonprindiville** [comment]
> I know that the new code view has its own find experience, but it's often not what I want

Yeah, the new half-baked "find-in-page" tool is a hindrance, IMO. This sucks.

It sucks because as you mention: the entire source code is no longer in the DOM so the browser's find-in-page is worthless.

Even trying to use the GH-built "find" tool repeatedly kind of sucks: if I search for one term and then want to search for another, the second time I hit `command-F` I get dumped into the (now useless) browser find-in-page tool. If I want to use the JS find-in-page again then I have to `esc` before I `command-f` again? That's confusing and weird.

---

IMO re-implementing browser features in client-side JS sucks in general. This feels like a weird half-complete attempt that should go away. Maybe don't break browser features that work just fine?

**Comment by ndevenish** [comment]
this is still really annoying. I keep running into it in "blame" views. I end up here a lot because I want to open a file, find the blame, then step back to the previous version of the file for that line. Except now all the line numbers have changed so I have to search.
